deeds-dapp-webapp/src/main/webapp/vue-app/main.js

```diff
@@ -24,6 +24,7 @@ import * as exchange from './js/exchange.js';
 import * as authentication from './js/authentication.js';
 import * as tenantManagement from './js/tenantManagement.js';
 import * as deedMetadata from './js/deedMetadata.js';
+import * as deedService from './js/deedService.js';
```

Comment:
Add a significant name of service that is similar to REST Service name : 

```suggestion
import * as tokenMetricService from './js/tokenMetricService.js';
```

Reply:
done